internal/factory/container/container.go

```diff
@@ -212,12 +213,12 @@ func (c *container) SpecAddAnnotations(ctx context.Context, sb SandboxIFace, con
 		}
 	}
 
-	c.spec.AddAnnotation(annotations.Image, userRequestedImage)
-	imageName := ""
+	c.spec.AddAnnotation(annotations.UserRequestedImage, userRequestedImage)
+	someNameOfThisImage := ""
```

Comment:
https://github.com/cri-o/cri-o/pull/8590#discussion_r1754581987 :
> I'm not personally a huge fan of how verbose this variable is. What is the risk to cri-o programmers if we leave the name as-is? do the comments present not document that risk?
>
> ---
> The risk has materialized in [#8212 (comment)](https://github.com/cri-o/cri-o/pull/8212#discussion_r1734986296) / [#8212 (comment)](https://github.com/cri-o/cri-o/pull/8212#discussion_r1747337067) . (And I don’t know what [#8590 (comment)](https://github.com/cri-o/cri-o/pull/8590#discussion_r1752967914) does.)
> 
> I don’t know that these ugly verbose names would _help_ but also I don’t have any better ideas.
> 
> ---
> Ideally, perhaps, CRI-O would change the logic to record some other values that are more known to be relevant (e.g. `PullImage` now returns a pretty well-specified repo@digest value). But that’s not this PR, and it would almost certainly change the API outputs; so I’ll leave the CRI-O maintainers to navigate that.
> 
> The ugly verbose names might serve as a gentle encouragement :)